$ OpenBSD: $

We don't need to build ALSA/Pulseaudio-related code because the 
sound on OpenBSD in tdesktop works through openal-soft (via sndio).
At the same time, we should not get an error about an unsupported OS. 

Index: Telegram/ThirdParty/libtgvoip/audio/AudioOutput.cpp
--- Telegram/ThirdParty/libtgvoip/audio/AudioOutput.cpp.orig
+++ Telegram/ThirdParty/libtgvoip/audio/AudioOutput.cpp
@@ -37,6 +37,7 @@
 #include "../os/linux/AudioOutputPulse.h"
 #include "../os/linux/AudioPulse.h"
 #endif
+#elif defined(__OpenBSD__)
 #else
 #error "Unsupported operating system"
 #endif
